quirks/change-password-URLs.json

```diff
@@ -6,6 +6,7 @@
     "ea.com": "https://myaccount.ea.com/cp-ui/security/index",
     "fnac.com": "https://secure.fnac.com/account/update-password",
     "geocaching.com": "https://www.geocaching.com/account/settings/changepassword",
+    "github.com": "https://github.com/settings/security",
```

Comment:
Github has adopted the Well-Known URL for Changing Passwords, so there’s no need to add it:
http://github.com/.well-known/change-password

Reply:
thanks